src/commands/dev/mod.rs

```diff
@@ -84,6 +85,18 @@ pub fn dev(
     })
 }
 
+fn print_alpha_warning() {
+    println!(
+        "{} wrangler dev is currently unstable and there are likely to be breaking changes.",
+        emoji::WARN
+    );
+    println!(
+        "{} For this reason, we cannot yet recommend using wrangler dev for integration testing.",
```

Comment:
```suggestion
        "{} We do not recommend wrangler dev for integration testing.",
```

Reply:
I don't think I want to change this because we want people to use this for testing in the future, just not right now.

Reply:
Ok there are several other ways to remove words without removing info. e.g. 
```suggestion
        "{} We do not recommend using wrangler dev for integration testing yet.",
```